src/Models/Location.php

```diff
@@ -142,205 +157,41 @@ public function getSelectorTitleAttribute()
-        }
-
-        return "/{$this->market->slug}/book-online/{$this->slug}";
-    }
-
-    public function getReservationsPathAttribute()
-    {
-        if ($this->market->locations_count === 1) {
-            return "/{$this->market->slug}/reservations";
-        }
-
-        return "/{$this->market->slug}/reservations/{$this->slug}";
-    }
-
     public function getPhoneLinkAttribute()
     {
-        return 'tel:' . preg_replace("/[^0-9]/", "", $this->phone);
+        return 'tel:' . preg_replace("/[^0-9]/", "", $this->phone()->full_number);
     }
 
     public function getDirectionsUrlAttribute()
     {
         return 'https://www.google.com/maps/dir/+Your+location/+' . Str::of($this->title)->replace(' ', '+') . ',@' . $this->latitude . ',' . $this->longitude;
     }
 
-    public function getBookingsYesterdayAttribute()
-    {
-        /** @var OrderInterface $service */
-        $service = findService(OrderInterface::class);
-        if ($service) {
-            return $service::itemFilter()
-                ->bySellableType(morphClass('booking') ?? '')
-                ->byLocation($this->id)
-                ->yesterday()
-                ->apply()
-                ->count();
-        }
-
-        return 0;
-    }
-
-    public function getRevenueBookedYesterdayAttribute()
-    {
-        /** @var OrderInterface $service */
-        $service = findService(OrderInterface::class);
-        if ($service) {
-            $amount = $service::itemFilter()
-                ->bySellableType(morphClass('booking') ?? '')
-                ->byLocation($this->id)
-                ->yesterday()
-                ->apply()
-                ->sum(function (OrderItemInterface $orderItem) {
-                    return $orderItem->getAmountTotal()->getDiscountedAmount();
-                });
-
-            return number_format($amount / 100, 2);
-        }
-
-        return 0;
-    }
-
-    public function getBookingsLastWeekAttribute()
-    {
-        /** @var OrderInterface $service */
-        $service = findService(OrderInterface::class);
-        if ($service) {
-            return $service::itemFilter()
-                ->bySellableType(morphClass('booking') ?? '')
-                ->byLocation($this->id)
-                ->week()
-                ->apply()
-                ->count();
-        }
-    }
-
-    public function getRevenueBookedLastWeekAttribute()
-    {
-        /** @var OrderInterface $service */
-        $service = findService(OrderInterface::class);
-        if ($service) {
-            $amount = $service::itemFilter()
-                ->bySellableType(morphClass('booking') ?? '')
-                ->byLocation($this->id)
-                ->week()
-                ->apply()
-                ->sum(function (OrderItemInterface $orderItem) {
-                    return $orderItem->getAmountTotal()->getDiscountedAmount();
-                });
-
-            return number_format($amount / 100, 2);
-        }
-    }
-
```

Comment:
@pdbreen Could you add these 4 reporting functions to the checkout package and allow me to pass a location to them? They are currently used in email reports.

Reply:
These are already implemented by the checkout package -- not that they are using the `OrderInterface::class` to provide an answer, but only when that service is available.  Not sure what more you want done.

Reply:
I added it back. Thank you!